app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -70,16 +69,6 @@ def upload_pdf_and_attach_special_issue(form526_submission, pdf)
       end
     end
 
-    def filtered_bp_readings(bp_readings)
-      bp_readings = bp_readings.filter do |reading|
-        reading[:issued].to_date > 1.year.ago
-      end
-
-      bp_readings.sort_by do |reading|
-        reading[:issued].to_date
-      end.reverse!
-    end
-
     def filtered_medications(medication_request_response)
```

Comment:
could we also move this logic the way you removed the `filtered_bp_readings`

Reply:
addressed in 8798d3e